yarn-project/deploy_npm.sh

```diff
@@ -81,6 +81,7 @@ deploy_package l1-artifacts
 deploy_package ethereum
 deploy_package noir-compiler
 deploy_package noir-contracts
+deploy_package noir-protocol-circuits
```

Comment:
I **think** this should be after `merkle-tree` since noir-protocol-circuits depends on that

Reply:
ah yes correct, I didn't look at `devDependencies`